independent-publisher-connectors/JiraSearch/apiDefinition.swagger.json

```diff
@@ -0,0 +1,392 @@
+{
+  "swagger": "2.0",

```

Comment:
@paulculmsee There is another PR open for this connector, Is it a different connector? If not, please remove these connector files from the PR